### PR TITLE
SNS support

### PR DESCRIPTION
**Pull request by 13shivam**
- sns integration and unit test added
- demo app updated 

Changes are verified with Jaeger local setup.

### CODE REVIEW

**Review by Sunn-y-Arora — APPROVED** [review]
LGTM

**Review by Sunn-y-Arora — CHANGES_REQUESTED** [review]
can u pls add the docs as well 
https://github.com/razorpay/opencensus-php/tree/master/docs/content

File: docs/content/integrating-sns.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+To add Spans
+
+```php
+<?php
+use Aws\Sqs\SqsClient;
+
+
+        ...
+        Tracer::inSpan(['name' => 'SNS:example'], function () {
```
Comment (jitendra-1217) [inline docs/content/integrating-sns.md:31]: Need this wrapper line?

Comment (13shivam) [reply docs/content/integrating-sns.md:31]: yes, this is to tell client ops should be under a span

**Review by Sunn-y-Arora — APPROVED** [review]

**Pull request merged**